test/testBearingRange2D.jl

```diff
@@ -314,12 +314,12 @@ end
 ##
 
 # Start with an empty graph
-N = 1
+N = 75
 fg = initfg()
 
 # Add landmark
 addVariable!(fg, :l1, Point2, tags=[:LANDMARK;])
-addFactor!(fg, [:l1], PriorPoint2(MvNormal([20.0;0.0], Matrix(Diagonal([1.0;1.0].^2)))),  graphinit=false ) # could be IIF.Prior
+addFactor!(fg, [:l1], PriorPoint2(MvNormal([20.0;0.0], Matrix(Diagonal([0.1;0.1].^2)))),  graphinit=false ) # could be IIF.Prior
```

Comment:
landmark  at (0,0)

Reply:
Jip, I saw the change was not needed as it's only a single particle on l1. 
I just needed a tighter covariance to get enough density to test for reliably.